package.json

```
@@ -35,20 +34,17 @@
     "test": "pnpm run-on-packages test && pnpm test:unit",
     "test:unit": "pnpm run with-dev-env vitest",
     "test:watch": "pnpm test:unit --watch",
-    "test:e2e": "pnpm run db:delete-saved-replies && pnpm run with-dev-env playwright test",
-    "test:e2e:clean": "pnpm run db:delete-saved-replies && pnpm run with-dev-env playwright test",
-    "test:e2e:with-data": "pnpm run with-dev-env playwright test",
-    "test:e2e:debug": "pnpm run db:delete-saved-replies && pnpm run with-dev-env playwright test --debug",
-    "test:e2e:headed": "pnpm run with-dev-env playwright test --headed",
-    "test:e2e:headed:clean": "pnpm run db:delete-saved-replies && pnpm run with-dev-env playwright test --headed",
-    "test:e2e:setup": "pnpm run with-dev-env playwright install",
+    "test:e2e": "bash scripts/test.sh playwright test",
+    "test:e2e:debug": "bash scripts/test.sh playwright test --debug",
+    "test:e2e:headed": "bash scripts/test.sh playwright test --headed",
     "typecheck": "pnpm run with-dev-env pnpm run build && pnpm run-on-packages typecheck && pnpm run with-dev-env tsc --noEmit && pnpm run db:check",
     "services:start": "pnpm with-dev-env pnpm supabase start && (docker ps | grep helperai-nginx || pnpm nginx:start)",
     "services:stop": "pnpm supabase stop && docker stop helperai-nginx || true",
     "nginx:start": "docker run -d --rm --name helperai-nginx -p 80:80 -p 443:443 --add-host host.docker.internal:host-gateway -v \"$(pwd)/scripts/docker/local-nginx/helperai_dev.conf:/etc/nginx/conf.d/default.conf\" -v \"$(pwd)/scripts/docker/local-nginx/certs:/etc/ssl/certs\" nginx:1.27.5",
     "generate-ssl-certificates": "bash scripts/generate-ssl-certificates.sh",
     "run-on-packages": "pnpm run --filter=!helperai --if-present --workspace-concurrency=1",
-    "with-dev-env": "dotenv -e ./.env.development.local -e ./.env.local --"
+    "with-dev-env": "dotenv -e ./.env.development.local -e ./.env.local --",
+    "with-test-env": "dotenv -e ./.env.test.local --"
```

```suggestion
    "with-dev-env": "dotenv -e ./.env.development -e ./.env.development.local -e ./.env.local --",
    "with-test-env": "dotenv -e ./.env.test -e ./.env.test.local --"
```

I think we need these?